tests/core/test_policies.py

```diff
@@ -394,86 +397,136 @@ def test_normalization(
     async def test_gen_batch(self, trained_policy: TEDPolicy, default_domain: Domain):
         training_trackers = await train_trackers(default_domain, augmentation_factor=0)
         interpreter = RegexInterpreter()
-        training_data, label_ids = trained_policy.featurize_for_training(
+        training_data, label_ids, entity_tags = trained_policy.featurize_for_training(
             training_trackers, default_domain, interpreter
         )
         label_data, all_labels = trained_policy._create_label_data(
             default_domain, interpreter
         )
         model_data = trained_policy._create_model_data(
-            training_data, label_ids, all_labels
+            training_data, label_ids, entity_tags, all_labels
         )
         batch_size = 2
+        print(len(next(model_data._gen_batch(batch_size=batch_size))))
+        print()
+        for k, v in model_data.items():
+            print(k)
+            for _k, _v in v.items():
+                print("  ", _k)
+                for __v in _v:
+                    print("    ", __v.shape)
```

Comment:
```suggestion
```

Reply:
oops 😂

Reply:
Sorry, I just saw the whole block can be removed and not just the print statements. Just pointing out since I can't edit the suggestion.

Reply:
it's ok, thanks, I'll remove